Emby.Naming/TV/SeriesResolver.cs

```diff
@@ -0,0 +1,64 @@
+using System.IO;
+using System.Text.RegularExpressions;
+using Emby.Naming.Common;
+
+namespace Emby.Naming.TV
+{
+    /// <summary>
+    /// Used to resolve information about series from path.
+    /// </summary>
+    public class SeriesResolver
```

Comment:
This class is better suited as a static class. There's no state and only a single public method.

Reply:
Fixed